docs/admins/membership-agreements.rst

```diff
@@ -0,0 +1,41 @@
+:og:description: Apiary requires prospective members to review and sign a membership agreement before paying dues.
+
+Membership agreements
+=====================
+
+.. vale write-good.E-Prime = NO
+.. vale write-good.Weasel = NO
+
+Apiary requires prospective members to review and sign a membership agreement before paying dues. This feature is currently implemented using `DocuSign Embedded Signing <https://developers.docusign.com/docs/esign-rest-api/esign101/concepts/embedding/>`_.
+
+Parent or legal guardian signatures
+-----------------------------
+
+By default, the agreement only collects a signature from the member themselves.
+For members considered a minor by the State of Georgia, a parent or legal guardian must co-sign the agreement.
+
+This feature requires an administrator to enter the parent or legal guardian's name and email address within the administrative interface.
+
+.. vale Google.Parens = NO
+.. vale Google.WordList = NO
+
+#. From the Apiary homepage, click the :guilabel:`Admin` link in the top navigation bar.
+#. Under the :guilabel:`Other` header in the left sidebar, click :guilabel:`Users`.
+#. Search for the user, then click the edit button (pencil icon |editicon|) on their row.
+#. Scroll to the :guilabel:`Parent or Guardian Signature` section, then enter **both** the :guilabel:`Parent or Guardian Name` **and** :guilabel:`Parent or Guardian Email`.
+#. Scroll to the bottom of the page, then click :guilabel:`Update User`.
+
+.. vale Google.Passive = NO
+.. vale Google.Will = NO
+.. vale write-good.E-Prime = NO
+.. vale write-good.Passive = NO
+
+.. important::
```

Comment:
I don't love the formatting here, but it legitimately is important to read, so I wanted it to stand out a bit.